indigo_app/forms/works.py

```diff
@@ -1414,8 +1412,8 @@ def facet_taxonomy(self, taxonomy_tree, qs):
         qs = self.filter_queryset(qs, exclude="taxonomy_topic")
         # count works per taxonomy topic
         counts = {
-            x["taxonomy_topics__slug"]: x["count"]
-            for x in qs.values("taxonomy_topics__slug").annotate(count=Count("taxonomy_topics__slug")).order_by()
+            x: qs.filter(taxonomy_topics__slug=x).count()
```

Comment:
this will do a new query for every taxonomy slug, which is too slow.

In what cases is the previous mechanism incorrect?

Reply:
I think all cases, e.g. https://edit.laws.africa/places/za/works?taxonomy_topic=projects-priority-1-highest-recent-updates
The count in the facet is 250 but the works listed are 181
<img width="981" alt="image" src="https://github.com/user-attachments/assets/0f897c32-21f1-4e9c-be56-eb91b3786f56" />